relay-event-schema/src/protocol/span.rs

```diff
@@ -718,6 +722,34 @@ impl Getter for SpanData {
     }
 }
 
+/// A link from a span to another span.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+pub struct SpanLink {
+    /// The trace id of the linked span
+    #[metastructure(required = true, trim = false)]
+    pub trace_id: Annotated<TraceId>,
+
+    /// The span id of the linked span
+    #[metastructure(required = true, trim = false)]
+    pub span_id: Annotated<SpanId>,
+
+    /// The parent span id of the linked span
+    #[metastructure(trim = false)]
+    pub parent_span_id: Annotated<SpanId>,
+
+    /// Whether the linked span was positively/negatively sampled
+    #[metastructure(trim = false)]
+    pub sampled: Annotated<bool>,
+
+    /// Span link attributes, similar to span attributes/data
+    #[metastructure(pii = "maybe", trim = false)]
+    pub attributes: Annotated<Object<Value>>,
```

Comment:
Is this the correct way to let Relay know that this field can contain PII and should be scrubbed?

Reply:
To apply default scrubbing rules (not just when the user explicitly configures it through advanced rules), set `pii = "true"`.

Reply:
Thanks! As discussed offline, let's stick with "maybe"